src/controller.cpp

```diff
@@ -16,17 +17,20 @@ void Controller::configure(
   costmap_ros_ = costmap_ros;
   tf_buffer_ = tf;
   name_ = name;
+  parameters_handler_ = std::make_unique<ParametersHandler>(parent);
 
   auto node = parent_.lock();
+  clock_ = node->get_clock();
   // Get high-level controller parameters
   auto getParam = utils::getParamGetter(node, name_);
   getParam(visualize_, "visualize", false);
 
   // Configure composed objects
-  optimizer_.initialize(parent_, name_, costmap_ros_);
+  optimizer_.initialize(parent_, name_, costmap_ros_, parameters_handler_.get());
   path_handler_.initialize(parent_, name_, costmap_ros_, tf_buffer_);
   trajectory_visualizer_.on_configure(parent_, costmap_ros_->getGlobalFrameID());
 
+  parameters_handler_->start();
```

Comment:
Should happen in `activate` instead of `configure` (just lifecycle convention)

Reply:
done